maintenance/scripts/glossary_template.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+  <head>
+      <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+      <meta name="generator" content="PhpSpreadsheet, https://github.com/PHPOffice/PhpSpreadsheet">
+      <title>Untitled Spreadsheet</title>
+      <meta name="author" content="Unknown Creator" />
+      <meta name="title" content="Untitled Spreadsheet" />
+      <meta name="company" content="Microsoft Corporation" />
```

Comment:
```suggestion
      <meta name="author" content="IOF Maintenance Ontology Working Group" />
      <meta name="title" content="IOF MRO Glossary" />
      <meta name="company" content="IOF/OAGi" />
```
I would make this match what is in the ontology file. Either define it here or replace it with the script. Just a suggestion.